packages/video_player/video_player_avfoundation/darwin/Classes/FVPVideoPlayerPlugin.m

```diff
@@ -327,6 +341,83 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem *)item
   return self;
 }
 
+/**
```

Comment:
We've now standardized on `///`-style declaration comment in our Obj-C code, so it should be used instead of `/** */`